R/rm.ridge.R

```diff
@@ -1,31 +1,57 @@
+#' @description
+#' Computes retention time intervals of a selected retention time range.
+#' @param rt full retention time vector.
+#' @param intensity Intensity vector.
+#' @param rt_range Selected retention time range vector.
+#' @return A list object:
+#' \itemize{
+#'   \item over_rt - upper retention time interval
+#'   \item under_rt - lower retention time interval
+#'   \item within_rt - intermediate retention time interval
+#' }
+#' @export
+compute_rt_intervals <- function(rt, intensity, rt_range){
+    rt_max <- max(rt[rt_range])
+    rt_min <- min(rt[rt_range])
+
+    over_rt <- which(rt > rt_max)
+    under_rt <-  which(rt < rt_min)
+    within_rt <- which(between(rt, rt_min, rt_max))
+
+    rt_range <- new("list")
+    rt_range$over_rt <- over_rt
+    rt_range$under_rt <- under_rt
+    rt_range$within_rt <- within_rt
+    return(rt_range)
+}
```

Comment:
`intensity` variable is not needed in this function.

The return can be put into one statement, also overriding a variable inside a function (`rt_range`) is not good.

Reply:
Corrected ([1a6eb70](https://github.com/RECETOX/recetox-aplcms/pull/171/commits/1a6eb702291e221f99c93346961f0b2347ef8563))